quaint/src/connector/postgres/url.rs

```diff
@@ -435,43 +431,6 @@ pub(crate) struct PostgresUrlQueryParams {
     pub(crate) ssl_mode: SslMode,
 }
 
-// A SearchPath connection parameter (Display-impl) for connection initialization.
-struct CockroachSearchPath<'a>(&'a str);
-
-impl Display for CockroachSearchPath<'_> {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        f.write_str(self.0)
-    }
-}
-
-// A SearchPath connection parameter (Display-impl) for connection initialization.
-struct PostgresSearchPath<'a>(&'a str);
-
-impl Display for PostgresSearchPath<'_> {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        f.write_str("\"")?;
-        f.write_str(self.0)?;
-        f.write_str("\"")?;
-
-        Ok(())
-    }
-}
-
-// A SetSearchPath statement (Display-impl) for connection initialization.
-struct SetSearchPath<'a>(Option<&'a str>);
-
-impl Display for SetSearchPath<'_> {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        if let Some(schema) = self.0 {
-            f.write_str("SET search_path = \"")?;
-            f.write_str(schema)?;
-            f.write_str("\";\n")?;
-        }
-
-        Ok(())
-    }
-}
-
```

Comment:
cool but spooky that this is just all gone 👀

Reply:
Huh, why is that gone? This is still used https://github.com/prisma/prisma-engines/blob/476ee04d9837606b5d38917a83f4559d9b04f24c/quaint/src/connector/postgres/native/mod.rs#L136-L155

Reply:
We just had two types with the same name. The ones from the snippet provided by Flavian are defined in `/Users/laplab/Desktop/prisma-engines/quaint/src/connector/postgres/native/mod.rs`.

Reply:
Ahh, right 👍